packages/app/client/src/components/projects/overview/modals/TeamMembersModal.js

```diff
@@ -0,0 +1,296 @@
+import React, { useState } from 'react'
+import CommonModal from '../../../../ui/atoms/CommonModal'
+import Typography from '@material-ui/core/Typography'
+import InputAdornment from '@material-ui/core/InputAdornment'
+import makeStyles from '@material-ui/core/styles/makeStyles'
+import { addMembersToProject } from '../../../../actions/projectActions'
+import { refreshToken } from '../../../../actions/authActions'
+import { Button, SearchBar, TextField } from '../../../../ui'
+import { compose } from 'recompose'
+import { connect } from 'react-redux'
+import { getFilteredUsers as getFilteredUsersActions } from '../../../../actions/authActions'
+import { Email } from '../assets'
+import { OutlinedInput } from '@material-ui/core'
+
+const useDefaultStyles = makeStyles(theme => ({
+  wrapper: {
+    padding: '5%',
+    textAlign: 'center'
+  },
+  header: {
+    textAlign: 'center',
+    fontWeight: 'bold'
+  },
+  descriptionText: {
+    margin: '5% 0',
+    fontSize: 18,
+    textAlign: 'center'
+  },
+  usersList: {
+    borderBottom: '1.6px solid #E7E7E7',
+    width: '90%',
+    maxHeight: '207px'
+  },
+  personInfo: {
+    borderBottom: '1.6px solid #E7E7E7',
+    height: '69px',
+    display: 'flex',
+    width: '100%',
+    justifyContent: 'space-between',
+    alignItems: 'center',
+    marginLeft: '5%'
+  },
+  personFullName: {
+    fontSize: '17px',
+    marginBottom: '2%',
+    textAlign: 'left'
+  },
+  avatar: {
+    width: '53px',
+    height: '53px',
+    borderRadius: '50%',
+    marginRight: '2%'
+  },
```

Comment:
I saw this avatar style before. I think it will be used in others features. We could make it as a system reusable component in our atoms folder, what do you think?

Reply:
sounds create, also we can do personalInfo like UI kit(avatar+fullname+button)

Reply:
did it